### PR TITLE
11.0 mig purchase open qty

### PR DESCRIPTION
**Pull request by andreasdsp**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash together adjacent "OCA Transbot.." commits using https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests, and you can do the same on some of the initial commits by @mreficent, preserving the logical sets.

File: purchase_open_qty/README.rst [hunk 1]
```diff
@@ -16,18 +16,7 @@ To use this module, you need to go to Purchase Orders and select the new filters
...
-===========
-
-Bugs are tracked on `GitHub Issues
-<https://github.com/OCA/purchase-workflow/issues>`_. In case of trouble, please
-check there if your issue has already been reported. If you spotted it first,
-help us smash it by providing detailed and welcomed feedback.
-
-Credits
-=======
+   :target: https://runbot.odoo-community.org/runbot/142/11.0
```
Comment (LoisRForgeFlow) [inline purchase_open_qty/README.rst]: Check the [OCA template](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst) and adapt this README file to the smaller pieces at the (new) `readme` directory

**Comment by LoisRForgeFlow** [comment]
@andreasdsp Are you going to continue this migration?

**Review by hieulucky111 — APPROVED** [review]

**Review by CasVissers-360ERP — APPROVED** [review]
LGTM (Functional)

**Comment by LoisRForgeFlow** [comment]
No answer from @andreasdsp. 

@mreficent could you please supersede this PR attending Pedro's comments?

**Comment by MiquelRForgeFlow** [comment]
@lreficent done in https://github.com/OCA/purchase-workflow/pull/602.

**Comment by LoisRForgeFlow** [comment]
@pedrobaeza could you close this one?

**Pull request closed without merging**